controllers/device_syncer.go

```diff
@@ -153,8 +150,10 @@ func (ds *DeviceSyncer) findDiffDevice(
 		ed := edgeDevices[i]
 		edName := util.GetEdgeDeviceName(&ed, EdgeXObjectName)
 		if _, exists := kubeDevices[edName]; !exists {
+			klog.V(5).Infof("found redundant edge device %s", edName)
```

Comment:
How about printing all redundantEdgeDeviceNames, redundantKubeDeviceNames or syncedDeviceNames as a list? In this way, relevant information can be displayed in three logs, reducing the number of redundant log entries.

Reply:
Printing all redundantEdgeDeviceNames, redundantKubeDeviceNames or syncedDeviceNames as list in one line will make this log entry very long, may be hard to read.

Reply:
look good to me.